Documentation/ApiOverview/Events/Events/Extbase/Persistence/ModifyQueryBeforeFetchingObjectCountEvent.rst

```diff
@@ -0,0 +1,21 @@
+..  include:: /Includes.rst.txt
+..  index:: Events; ModifyQueryBeforeFetchingObjectCountEvent
+..  _ModifyQueryBeforeFetchingObjectCountEvent:
+
+=========================================
+ModifyQueryBeforeFetchingObjectCountEvent
+=========================================
+
+..  versionadded:: 14.0
+    The event :php:`_ModifyQueryBeforeFetchingObjectCountEvent` was
+    introduced
```

Comment:
```suggestion
..  versionadded:: 14.0
```

You can use an empty versionadded tag to just display when the event was added. We only use the text if there is more information like "to replace hook xy"

Reply:
OK. Done